libraries/AP_NMEA_Output/AP_NMEA_Output_config.h

```diff
@@ -0,0 +1,22 @@
+#include <AP_HAL/AP_HAL_Boards.h>
+
+// Needs SerialManager + (AHRS or GPS)
+
+#ifdef HAL_BUILD_AP_PERIPH
+    #define AP_NMEA_OUTPUT_LIBRARY_REQUIREMENTS (defined(HAL_GCS_ENABLED) && HAL_GCS_ENABLED && defined(HAL_PERIPH_ENABLE_GPS))
+
+    #ifndef HAL_NMEA_OUTPUT_ENABLED
+    #define HAL_NMEA_OUTPUT_ENABLED (!HAL_MINIMIZE_FEATURES && AP_NMEA_OUTPUT_LIBRARY_REQUIREMENTS)
```

Comment:
Would be nice to put the HAL_MINIMIZE_FEATURES into `minimal.inc`

Reply:
in another PR...